assets/src/common/components/featured-image-select-media-frame.js

```diff
@@ -72,7 +72,7 @@ const FeaturedImageToolbarSelect = wp.media.view.Toolbar.Select.extend( {
 		const minWidth = state.collection.get( 'library' ).get( 'suggestedWidth' );
 		const minHeight = state.collection.get( 'library' ).get( 'suggestedHeight' );
 
-		if ( ! attachment || ( attachment.get( 'width' ) >= minWidth && attachment.get( 'height' ) >= minHeight ) ) {
+		if ( ! attachment || ! attachment.get( 'width' ) || ( attachment.get( 'width' ) >= minWidth && attachment.get( 'height' ) >= minHeight ) ) {
```

Comment:
This should fix the issue in https://github.com/ampproject/amp-wp/pull/2711#issuecomment-507504228

This could be a problem if `attachment.get( 'width' )` is `0`, but I'm not sure if or how often an image would have a width of `0`.